.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+<!--
+Thanks for opening a pull request!
+
+Before you continue, please make sure that you have read and understood the contribution guidelines, otherwise your changes may be rejected:
+https://github.com/streamlink/streamlink/blob/master/CONTRIBUTING.md#contributing-to-streamlink
+
+If possible, run the tests, code linting tools and documentation builds locally on your system first to avoid unnecessary build failure surprises.
+
+Also don't forget to add a meaningful description of your changes, so that the reviewing process is as simple as possible for the maintainers.
```

Comment:
Should we add a link to an example of a good PR here?

Reply:
No, I'll add something to the docs which can then be linked. This will partially solve #3195